src/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/Areas/Identity/Controllers/ManageController.cs

```diff
@@ -10,14 +10,15 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
-using Company.WebApplication1.Models;
-using Company.WebApplication1.Models.ManageViewModels;
-using Company.WebApplication1.Services;
+using Company.WebApplication1.Identity.Models;
+using Company.WebApplication1.Identity.Models.ManageViewModels;
+using Company.WebApplication1.Identity.Services;
 
-namespace Company.WebApplication1.Controllers
+namespace Company.WebApplication1.Identity.Controllers
 {
     [Authorize]
-    [Route("[controller]/[action]")]
+    [Area("Identity")]
+    [Route("tfp/[area]/[controller]/[action]")]
```

Comment:
What's `tfp`?

Reply:
Its just an acronym that we need for the MSAL libraries to work when we add Identity as a Service on top of Identity. It'll likely go away in the future. It stands for Trust framework policy.

Reply:
OK.

I'm not hugely concerned, but does it have to be exactly `tfp`? Couldn't MSAL use the `identity` prefix from the area instead?

Reply:
Sadly for now, the answers is no. But I'm going to work with Dan to see if they can make their clients more flexible

Reply:
OK, thanks for clarifying.